get-started/graph-db-vs-nosql/graph-db-vs-nosql.adoc

```diff
@@ -10,43 +10,43 @@
 
 .Goals
 [abstract]
-This guide quickly explains how graph databases are related to other NoSQL databases.
+This guide quickly explains how graph databases are related to other NoSQL databases and how they differ.
 
 .Prerequisites
 [abstract]
-You need no prior knowledge of other NoSQL stores, although it might be helpful to have read the guide on link:/developer/get-started/graph-database[graph databases].
+You need no prior knowledge of other NoSQL stores, although it is helpful to have read the guide on link:/developer/get-started/graph-database[graph databases] and understand basic data modeling questions and concepts.
 
 [role=expertise]
 {level}
 
 toc::[]
 
+
+== NoSQL Databases
+
 Although unhelpfully named, the NoSQL ("Not only SQL") space brings together many interesting solutions offering different data models and database systems, each suitable for different use cases and shapes of data.
 
 With the advent of the NoSQL movement, the "one-size-fits-all" proposition of large relational systems was replaced by making conscious decisions about finding the right tool for the job.
```

Comment:
delete 'making': "...was replaced by ~~making~~ conscious decisions about finding the right tool for the job."